src/scss/components/optional/_cuisines-widget.scss

```diff
@@ -0,0 +1,136 @@
+/**
+* Components > Cuisines Widget
+* =================================
+*
+* Example: Homepage – Popular cuisines.
+*
+* The `c-cuisinesWidget` component is an optional mixin within Fozzie.
+* If you'd like to use it in your project you can include it by adding `@include cuisinesWidget();` into your SCSS dependencies file.
+*
+* Documentation:
+* https://fozzie.just-eat.com/styleguide/ui-components/cuisines-widget
+*/
+
+//TODO: Improvement of this solution: WH-682 as suggested in https://github.com/justeat/fozzie/pull/188#issuecomment-442138750
+
+$cuisinesWidget-titleColour: $white;
+$cuisinesWidget-defaultBackground: $brand--red;
+$cuisinesWidget-boxShadow: 0 2px 4px 0 rgba(0, 0, 0, 0.1);
+$cuisinesWidget-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0 rgba(0, 0, 0, 0.04);
+$cuisinesWidget-gradient: linear-gradient(to top, rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0));
+
+@mixin cuisinesWidget() {
+    @include media-context(('narrow-mid': 570px, 'mid-wide': 1240px)) {
+
+       .c-cuisinesWidget {
+           display: block;
+           border-radius: 8px;
+           overflow: hidden;
+           flex: none;
+           position: relative;
+           background-color: $cuisinesWidget-defaultBackground;
+           box-shadow: $cuisinesWidget-boxShadow;
+
+           &:hover {
+               box-shadow: $cuisinesWidget-hoverBoxShadow;
+           }
+       }
+
+       .c-cuisinesWidget-name {
+           position: absolute;
+           bottom: 0;
+           left: 0;
+           width: 100%;
+           padding: spacing(x2);
+           color: $cuisinesWidget-titleColour;
+           text-align: center;
+           background-image: $cuisinesWidget-gradient;
+
+           @include media('>=mid-wide') {
+               @include font-size(large, false);
+               padding: spacing(x3);
+           }
+       }
+
+       .c-cuisinesWidget-img {
+           vertical-align: bottom;
+           width: 100%;
+       }
+
+       // grid of three elements for desktop, two for tablets and one for mobile
```

Comment:
Best to refer to the breakpoints in this comment rather than desktop/mobile.  So:

`// grid of three elements for wide viewports, two for mid and one for narrow`

Reply:
✅